server/copy.go

```diff
@@ -0,0 +1,61 @@
+package main
```

Comment:
Something out of scope which just caught my eye. It should be `package server` right?

Reply:
Yeah, the whole package needs to be renamed.